app/src/main/java/org/oppia/app/topic/TopicActivity.kt

```diff
@@ -17,12 +17,17 @@ private const val TOPIC_ACTIVITY_TOPIC_ID_ARGUMENT_KEY = "TopicActivity.topic_id
 private const val TOPIC_ACTIVITY_STORY_ID_ARGUMENT_KEY = "TopicActivity.story_id"
 
 /** The activity for displaying [TopicFragment]. */
-class TopicActivity : InjectableAppCompatActivity(), RouteToQuestionPlayerListener,
-  RouteToStoryListener, RouteToExplorationListener, RouteToRevisionCardListener {
+class TopicActivity :
+  InjectableAppCompatActivity(),
+  RouteToQuestionPlayerListener,
+  RouteToStoryListener,
+  RouteToExplorationListener,
+  RouteToRevisionCardListener {
 
   private var internalProfileId: Int = -1
   private lateinit var topicId: String
   private var storyId: String? = null
+
   @Inject
   lateinit var topicActivityPresenter: TopicActivityPresenter
```

Comment:
Bring this to single line as per style guidelines
`@Inject lateinit var topicActivityPresenter: TopicActivityPresenter`

Reply:
Ktlint is a bit different, it says to be on the next line. Also, Ben discuss the same in yesterday's meeting

Reply:
Got it.